File: Templates/Java/src/main/java/sdkgenerator/person/Person.java
package sdkgenerator.person;

import com.google.gson.annotations.SerializedName;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public class Person {

    @NotNull
    @SerializedName("first_name")
    public String FirstName;

    @NotNull
    @SerializedName("last_name")
    public String lastName;

    @NotNull
    @SerializedName("age")
    public int age;

    @NotNull
    @SerializedName("address")
    public Address address;

    @NotNull
    @SerializedName("jobs")
    public List<Job> jobs;

    public @NotNull String getFirstName() {
        return FirstName;
    }

    public @NotNull String getLastName() {
        return lastName;
    }

    public int getAge() {
        return age;
    }

    public @NotNull Address getAddress() {
        return address;
    }

    public @NotNull List<Job> getJobs() {
        return jobs;
    }
}

File: SdkGenerator/Model/Example/PersonExample/Person.cs
using System.Text.Json.Serialization;

namespace SdkGenerator.Model.Example.PersonExample;

public class Person
{
    [JsonPropertyName("first_name")] public required string FirstName { get; set; }
    [JsonPropertyName("last_name")] public required string LastName { get; set; }
    [JsonPropertyName("age")] public required int Age { get; set; }
    [JsonPropertyName("address")] public required Address Address { get; set; }
    [JsonPropertyName("jobs")] public required List<Job> Jobs { get; set; }
    [JsonPropertyName("parent")] public Person? Parent { get; set; }
    [JsonPropertyName("child")] public Person? Child { get; set; }
}